<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1260</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1260</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="pltWidget" native="true">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>10</y>
     <width>1231</width>
     <height>801</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_plot">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>840</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Plot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>840</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
